service/start.go

```diff
@@ -13,27 +14,28 @@ import (
 	"github.com/spf13/viper"
 )
 
-// Start a service
-func (service *Service) Start() (serviceIDs []string, err error) {
+// Start starts the service.
+func (service *Service) Start() ([]string, error) {
```

Comment:
it's a good practice using named returns on Public methods for **basic** types. Otherwise returned values doesn't make sense to dev and there is no documentation about it.

Reply:
Done